catala/cantar/tastar/tastar.md

```diff
@@ -1 +1,5 @@
-Tastes una llaminadura en forma de cor, i a fora sembla que para de ploure. Obres la tenda i... tot és de sucre!
+Tastes una llaminadura en forma de cor, i a fora sembla que para de ploure. Obres la tenda i... tot és de sucre! El sol regalima lentament sobre l'horitzó gotes de caramel, i et preguntes quin gust deu tindre.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.